honeybee_radiance/cli/view.py

```diff
@@ -191,7 +195,12 @@ def merge_view(input_folder, base_name, extension, scale_factor, folder, name):
 
         # search for a single .vf in the folder and, if it's found, grab the info
         views = sorted(f for f in os.listdir(input_folder) if f.endswith('.vf'))
-        if len(views) == 1:  # replace the header with the info in the view
+        if view:
```

Comment:
We probably only want to do this when there are more than one view.

```suggestion
        if view and len(views) != 1:
```